sale_timesheet_ux/models/project_task.py

```diff
@@ -1,10 +1,12 @@
-from odoo import models
+from odoo import models, fields
 
 
 class ProjectTask(models.Model):
 
     _inherit = 'project.task'
 
+    order_id = fields.Many2one('sale.order', 'Sales Order', store=True, help="Sales order to which the task is linked.")
```

Comment:
@matiasperalta1 Mati, el build de runbot da un error que te dejo acá con este campo:
Two fields (order_id, sale_order_id) of project.task() have the same label: Sales Order. [Modules: sale_timesheet_ux and sale_timesheet]